<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggedIn</class>
 <widget class="QWidget" name="LoggedIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="viewEditDetailsBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View/Edit Your Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signOutBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createOrderBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>New Order</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="orderListTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>711</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Order_ID</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Order status</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delivery status</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>View Invoice</string>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="updateOrderTableBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>See current orders</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="updates_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>711</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>295</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Updates to Existing Orders</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Notifications:</string>
   </property>
  </widget>
  <widget class="QLabel" name="notification_count">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_order_updates">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="feedbackButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Leave feedback</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
